tests/Fixer/FunctionNotation/SingleLineThrowFixerTest.php

```diff
@@ -35,15 +35,18 @@ public function provideFixCases(): iterable
     {
         yield ['<?php throw new Exception; foo(
                     "Foo"
-                );'];
+                );',
+        ];
```

Comment:
yeah so fixed 50% , OK

Reply:
Why do you react this way? If there's something wrong with my comment I can hear it, why not simply say it?

Reply:
There is nothing wrong with your comment, you can say whatever you want to say about this PR.

We do see things different however and I put that in the comment, I see 50% improvement on something not directly related to what this PR is try to fix (which is a valid issue raised by end-user),
You see a reason to comment it as something that must be changed to be merged (otherwise, it was merged) and note absolutely nothing else about the PR.

You can set the bar to whatever you want for any PRs and comment on PRs not reaching that bar, it is all fair.
It is also fair for anyone, this time me as author, to not willing to put even more time and effort to reach that bar, but choose to move away from the PR instead.

>why not simply say it?

I did, here but also in different PRs, I also tried to get people together in monthly calls, in gitter, I tried to talk to a lot of people, proposed Trello boards, whatever to align effort, prioritize work, have meaningful talks about whatever related to the project, but did all people simple say what they want like just as you ask here from me? I know I did and I know what I got back.

Reply:
There is nothing wrong with your comment, you can say whatever you want to say about this PR.

You can set the bar to whatever you want for any PRs and comment on PRs not reaching that bar, it is all fair.
It is also fair for anyone, this time me as author, to not willing to put even more time and effort to reach that bar, but choose to move away from the PR instead.

Reply:
That is very true. However you could have just said that you didn't want to invest that time, maybe I was willing to make the changes myself (which I didn't have the time to do at the moment I made the review), or maybe I would have realized this is superfluous and merged the PR as-is.

Reply:
not sure if there is any deal here to be honest, let me reopen the PR as fix of crashing app is definitely nice to have!
#6653

Reply:
IMHO there shouldn't be **any** comments about formatting. This should be handled by CI jobs and either it passes and everything is OK, or it fails and it has to be fixed according to rules 🤷‍♂️ If something slips through and requires rules' upgrade, it's not concern of that PR and should be done in another PR. Let's focus on actual changes in the logic, let's don't waste time on discussions about brackets' placement 🙂 🙏

Reply:
half of Fixer project is about "brackets placement" 😅 
but I agree that if CI is not picking sth up, we could externalise from PR and have it as new-rule request, to non-block the PR ;)